packages/sdk38/src/encoding.ts

```diff
@@ -20,11 +20,6 @@ function sortJson(json: any): any {
   return result;
 }
 
-export function marshalTx(tx: StdTx): Uint8Array {
```

Comment:
This was never used internally. Interesting.

Nice to move simpler helpers to where they are used/needed